apps/content_filters/filter_condition/filter_condition_resource.py

```diff
@@ -75,3 +81,18 @@ class FilterConditionResource(Resource):
     privileges = {'POST': 'content_filters',
                   'PATCH': 'content_filters',
                   'DELETE': 'content_filters'}
+
+    def pre_request_post(self, request):
+        self._init_allowed_filters()
+
+    def pre_request_patch(self, request, lookup):
+        self._init_allowed_filters()
+
+    def _init_allowed_filters(self):
+        self.schema['field']['allowed'] = copy.copy(default_allowed_filters)
+        excluded_vocabularies = [vocabulary.strip() for vocabulary in
```

Comment:
but if that config is a string this handling to split and strip should be in one place and not duplicated